src/CatenaLogic.Analyzers/Analyzers/CL0002/CL0002Analyzer.cs

```diff
@@ -34,6 +36,12 @@ public override void HandleSyntaxNode(SyntaxNodeAnalysisContext context)
                 return;
             }
 
+            // Ignore Main method
+            if (string.Equals(methodSymbol.Name, DefaultMainMethodName) && methodSymbol.IsStatic)
```

Comment:
for performance reasons, I recommend checking IsStatic first. Remember this runs on every compilation and every thing you type in, so we need to make it as fast as possible.

Reply:
Great note